PotteryBarn/Core/Pieces/VanillaShop.cs

```diff
@@ -16,6 +16,14 @@ public static class VanillaShop {
               "Ashlands_floor_large",
               Workstation.Stonecutter,
               [ (Item.Grausten, 32) ])
+          .AddPiece(
+              "ashwood_arch_top",
+              Workstation.Workbench,
+              [(Item.Blackwood, 2)])
```

Comment:
I know it's cause of VS2022 autocorrect due to one element, but for clarity just re-add the spaces (after the [ and before ]).